docs/installation/dns_setup.md

```diff
@@ -1,157 +1,201 @@
+:::note
 
-***
-Epinio will install successfully even if your DNS setup is not complete. There is an exception to this,
-when letsencrypt issuer is used. Cert-Manager will fail to create certificates until the domain is accessible, due to
-the challenges that must be solved (read more about issuers [here](../howtos/other/certificate_issuers.md)). Even in that
-case though, after you finish your DNS setup, the challenges will be solved and Epinio should become functional but
-it may take a while before things balance out.
-***
+Epinio will install successfully even if your DNS setup is not complete.
+There is an exception when the Lets Encrypt issuer is used.
+Then `cert-manager` will fail to create certificates until the domain is accessible.
+You can read more about certificate issuers [here](../howtos/other/certificate_issuers.md).
+Epinio will work after the DNS setup is correct and the domain becomes available.
 
-To keep things simple, it is preferable that you finish your DNS setup before you install Epinio. To do so, you need
-to point your desired domain to the IP address of your Ingress controller. There are two steps in this process:
+:::
 
-1. Find the IP address of the ingress controller
-2. Setup your DNS
+For simplicity finish your DNS setup before installing Epinio.
+You need to point your desired domain to the IP address of your Ingress controller.
+The two steps in the process are described in the following sections:
 
-> **NOTE:** For development or demo environments, [an easy wildcard DNS setup can be used](wildcardDNS_setup.md) instead.
+1. [Find](#find-ip-for-ic) the IP address of the ingress controller
+1. [Configure](#config-dns) your DNS
 
-## Ingress controller IP address
+:::note
 
-Most Kubernetes clusters are running a "load balancer" service which is responsible of assigning
-IP addresses to load balanced services created on the cluster. Ingress controllers
-are such services (e.g. Traefik) and they should have an externally accessible IP address.
-This is needed for any Ingress resource to work. You can find the load balancer IP
-address of any service using `kubectl`. E.g.
+For development or demo environments, [an easy wildcard DNS setup can be used](wildcardDNS_setup.md).
+
+:::
+
+## Find the IP address of the ingress controller {#find-ip-for-ic}
+
+Most Kubernetes clusters run a "load balancer" service.
+It is responsible for assigning IP addresses to services, that are load balanced, created on the cluster.
+Ingress controllers are such services (for example, Traefik) so should have an externally accessible IP address.
+This is needed for any Ingress resource to work.
+
+You can find the load balancer IP address of any service using `kubectl`. For example:
 
 ```
 kubectl get svc -n kube-system traefik -o jsonpath={@.status.loadBalancer.ingress}
 ```
+<!-- TODO: I would like a sample output from the previous command showing the IP address field.-->
```

Comment:
On my local k3s this looks like
```
work@tagetarl:~/SUSE/dev/epinio> kubectl get svc -n kube-system traefik -o jsonpath={@.status.loadBalancer.ingress} ; echo
[map[ip:172.18.0.4]]
```

The `echo` in the command adds a newline for readability.
Note that this output is actually not JSON.
This is what go's `%+v` generates for the data structure.
As JSON it should have looked like
```
{ "ip": "172.18.0.4" }
```

Reply:
Thank you.